frontend/src/scenes/plugins/edit/PluginDrawer.tsx

```diff
@@ -177,6 +177,17 @@ export function PluginDrawer(): JSX.Element {
                                                 <UploadField />
                                             ) : fieldConfig.type === 'string' ? (
                                                 <Input />
+                                            ) : fieldConfig.type === 'choice' ? (
+                                                <Select
+                                                    defaultValue={fieldConfig.choices[0]}
```

Comment:
Just to be on the safe side, I'd add `choices?.[0]` since while the type of the object is an array, there's no guarantee what we'll get, or alternatively change the if to `fieldConfig.type === 'choice' && Array.isArray(fieldConfig.choices)`

This and plus a tests fails :)

Reply:
yeah the test is failing with a TS error related to the typing from plugin-scaffold, but that should be up-to-date?